ansible/configs/satellite-vm/post_software.yml

```diff
@@ -26,6 +26,17 @@
   roles:
     - role: satellite-verification
 
+- name: Set user info and data
+  hosts: localhost
+  tasks:
+    - name: Set satellite host name for the user
+      agnosticd_user_info:
+        data:
+          satellite_hostname: "{{ hostvars[groups['satellites'][0]]['publicname'] }}"
+          dns_zone: "{{ osp_cluster_dns_zone }}"
+          satellite_username: summit-admin
+          satellite_password: summit2020
```

Comment:
This is intentionally static, no point to change AgnosticV at this point and it would be neccesary not to be static, we just want to prepare it for the following usage of the config, so there would not need to be changes to `bookbag`.